pyproject.toml

```diff
@@ -1,23 +1,89 @@
+[build-system]
+requires = ["setuptools>=42", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+addopts = "--showlocals"
+testpaths = ["tests"]
+
+[tool.coverage.run]
+source = ["airbase"]
+
+[tool.coverage.report]
+skip_covered = true
+skip_empty = true
+show_missing= true
+
 [tool.black]
 line-length = 80
-target-version = ['py36','py37','py38','py39']
-include = '\.pyi?$'
-exclude = '''
-
-(
-  /(
-      \.eggs         # exclude a few common directories in the
-    | \.git          # root of the project
-    | \.hg
-    | \.mypy_cache
-    | \.tox
-    | \.venv
-    | _build
-    | buck-out
-    | build
-    | dist
-  )/
-  | foo.py           # also separately exclude a file named foo.py in
-                     # the root of the project
-)
-'''
+target-version = ["py37"]
+
+[tool.isort]
+line_length = 80
+py_version = "37"
+profile = "black"
+src_paths = ["airbase", "tests"]
+extend_skip = ["scripts"]
+
+[tool.mypy]
+warn_unused_configs = true
+warn_unused_ignores = false
+warn_no_return = true
+warn_return_any = true
+warn_unreachable = true
+show_error_codes = true
+pretty = true
+sqlite_cache = true
+exclude = "docs|scripts"
+
+[[tool.mypy.overrides]]
+module = ["tqdm", "aiohttp", "pytest", "aioresponses"]
+ignore_missing_imports = true
```

Comment:
There are stubs available for all of these except tqdm afaik - shouldn't we just add the stub packages to `extras_require:lint` in `setup.cfg`?

Reply:
I'm developing over Python 3.10 at the moment and mypy was complaining about missing stubs. I'll recreate my environment on Python 3.8 and check.